src/main/resources/sql/data.sql

```diff
@@ -36,4 +38,18 @@ INSERT INTO `comment` (`content`, `user_id`, `post_id`, `created_at`) VALUES ('H
 INSERT INTO `comment` (`content`, `user_id`, `post_id`, `created_at`) VALUES ('I wonder why the wisdom is always wise', 3, 5, '2008-03-04 02:14:59');
 
 -- Add dialogs between users
-INSERT INTO `dialog` (`created_at`, `last_update`) VALUES ('2008-01-27 14:39:07', '2008-02-10 16:42:00');
\ No newline at end of file
+INSERT INTO `dialog` (`created_at`, `last_update`) VALUES ('2008-01-27 14:39:07', '2008-02-10 16:42:00');
+INSERT INTO `dialog` (`created_at`, `last_update`) VALUES ('2009-01-27 14:39:07', '2009-02-10 16:42:00');
+INSERT INTO `dialog` (`created_at`, `last_update`) VALUES ('2009-01-27 14:39:07', '2009-02-10 16:42:00');
+
+-- Add connection between users and messages through dialog
+INSERT INTO `user_dialog` (`user_id`, `dialog_id`) VALUES (1, 2);
+INSERT INTO `user_dialog` (`user_id`, `dialog_id`) VALUES (2, 2);
+INSERT INTO `user_dialog` (`user_id`, `dialog_id`) VALUES (1, 3);
+INSERT INTO `user_dialog` (`user_id`, `dialog_id`) VALUES (3, 3);
+
+-- Add messages to dialog
+INSERT INTO `message` (`content`, `dialog_id`, `user_id`, `created_at`) VALUES ('hello', 2, 1, '2009-02-28 14:00:00');
+INSERT INTO `message` (`content`, `dialog_id`, `user_id`, `created_at`) VALUES ('nice to meet you', 2, 2, '2009-02-28 14:01:00');
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([plsql:S1192](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=118&issues=AWUpsvUnOTG07Bqb-YPx&open=AWUpsvUnOTG07Bqb-YPx)

[ISSUE_KEY]: AWUpsvUnOTG07Bqb-YPx